ceagle/api/v1/security.py

```diff
@@ -14,15 +14,20 @@
 #    under the License.
 
 import flask
+from flask import request  # noqa
 
+from ceagle.api import client
 
 security = flask.Blueprint("security", __name__)
 
+client = client.get_client("security")
```

Comment:
Yes, and this is much better because we know about this kind of misconfiguration early enough.